sdk/core/Azure.Core/src/Response.cs

```diff
@@ -86,9 +86,9 @@ public virtual BinaryData Content
         /// Indicates whether the message's <see cref="ResponseClassifier"/> considers this
         /// response an error.
         /// </summary>
-        public bool IsError { get; internal set; }
+        public virtual bool IsError { get; internal set; }
 
-        internal HttpMessageSanitizer? Sanitizer { get; set; }
+        internal HttpMessageSanitizer? Sanitizer { get; set; } = HttpMessageSanitizer.Default;
```

Comment:
Can we add a test that Mocked response works with the Exception ctor?

Reply:
There is one, but I can make it explicit